### DISCUSSION TITLE
Repositories starred by people not showing up

### DISCUSSION THREAD

**Discussion by aashutoshrathi** [community/community#66274] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When this is selected: 
<img width="812" alt="image" src="https://github.com/community/community/assets/21199234/d854f65a-5f4c-4cab-9926-af1185b9342b">

**Comment by guohengkai** [comment]
+1

**Comment by bassamanator** [comment]
People staring ⭐ my repos doesn't show up *anymore* when I goto https://github.com/. This was definitely the case before.

The last star listed on my feed is from 3 weeks ago, but I've been gaining stars.

The only `filters` not enabled are Sponsors, Recommendations, and Releases.

**Comment by hsiong** [comment]
the same question,  I have subscribed to this issue and am looking forward to a reply.

**Comment by sooftware** [comment]
Me too.
